README.md

```diff
@@ -5,93 +5,62 @@
 [![node](https://img.shields.io/badge/node-v9.0.0-green.svg)](https://nodejs.org/download/release/v9.0.0/)
 [![License](https://img.shields.io/badge/license-APACHE%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-> A distributed testing framework 
-
+> A distributed testing framework for hardware 
 ## Getting Started
 
-To set up Leviathan both hardware and software need to be configured. If you are setting up your standalone testbot, then please follow the instructions given below carefully before running tests on the Device Under Test (DUT). 
-
+To set up Leviathan both hardware and software need to be configured. If you are setting up your standalone testbot, then please follow the instructions given below carefully before running tests on the Device Under Test (DUT).
 
 ### Clone the repository
 
 - Clone this repository with `git clone --recursive` or   
 - Run `git clone` and then `git submodule update --init --recursive` to install submodules.
-
-
+- 
 ### Prerequisites needed
 
 - Install node and npm in your system. We recommend installing [LTS versions from NVM](https://github.com/nvm-sh/nvm#install--update-script).
 - Download the image you want to test on your DUT from [balena.io/os](https://balena.io/os#download).
 
-
 ### Configuration needed
 
 - Start building your standalone testbot by [following the guide](https://github.com/balena-io/testbot/blob/master/documentation/getting-started.md#quick-start-guide-for-testbot). 
-- Create your test configuration, by creating a `config.json` file in the `workspace` directory, following instructions mentioned in the [testbot docs](https://github.com/balena-io/testbot/blob/master/documentation/getting-started.md#run-your-first-test).
-- Move the image zip file you want to test with inside the `workspace` directory in Leviathan.
-- Extract the image, rename it to balena.img, and recompress the image into the `.gz` format. The final file would look like `balena.img.gz` file and has to be in the `./leviathan/workspace` directory. 
-
-
-### Packages needed for installation
+- Create your test configuration, by creating a `config.json` file in the `workspace` directory, following instructions mentioned in the [leviathan docs](https://github.com/balena-io/testbot/blob/master/documentation/getting-started.md#run-your-first-test).
```

Comment:
we should give the link to https://github.com/balena-os/leviathan/blob/master/documentation/Getting-started.md instead

Reply:
Good catch. That file has been broken down, since it was difficult to maintain. The part for quick start would be in `quickstart.md`